client/src/pages/admin/authorizationgroup/Show.js

```diff
@@ -25,23 +27,45 @@ export default class AuthorizationGroupShow extends Page {
 		setMessages(props,this.state)
 	}
 
+	getReportQueryPart(authGroupId) {
+		let reportQuery = {
```

Comment:
I propose to unversally adopt using `const` in JS and `final` in Java whenever the reference is not intended to change